python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -0,0 +1,164 @@
+import collections
+import datetime
+import sqlite3
+from lsst.pex.config import Config, ListField, ConfigurableField
+from lsst.pipe.base import Task, ArgumentParser
+from lsst.pipe.tasks.ingest import RegisterTask, ParseTask, RegisterConfig
+
+def _convertToDate(dateString):
+    """Convert a string into a date"""
```

Comment:
It's converting a string into a date object. "date" is ambiguous as it could just mean that it is finding the date in the string and returning the date as a string.